aspnet/fundamentals/localization.rst

```diff
@@ -203,7 +203,16 @@ If you only pass in one of the two (``culture`` or ``ui-culture``), the query st
 CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
+In old versions of ASP.NET a production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie.
```

Comment:
We do preserve the docs for previously released versions. For example, we have the RC1 docs at https://docs.asp.net/en/1.0.0-rc1/. But we don't want the docs for a specific version to read like a chronology of every change that we make along the way. 

Let's remove it for now. We can add back a note if we get a lot of request for this specific information.